app/models/user.rb

```diff
@@ -1,5 +1,7 @@
 class User < ApplicationRecord
     has_one :token
+    has_many :memberships
```

Comment:
you should change this to userCalender or fix the membership table to include user_id and calendar_id